application/templates/base.html

```diff
@@ -1,15 +1,10 @@
 <!DOCTYPE html>
-<!--[if lt IE 9]><html class="lte-ie8" lang="{{ html_lang|default('en') }}"><![endif]-->
-<!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
+<html lang="{{ html_lang|default('en') }}">
```

Comment:
Hang on, are you saying we've had no opening `<html>` tag in our pages... ever?

Reply:
Oh yeah, just checked prod website... not there :man_facepalming:

Reply:
Although W3 validator doesn't seem to mind...
https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.ethnicity-facts-figures.service.gov.uk%2F
:man_shrugging:

Reply:
@TheDoubleK it is there, it's just within the previous line... (the `<!--[if gt IE 8]><!-->` closes the comment, so the next bit is valid.